lib/hubspot/mailer.rb

```diff
@@ -60,8 +60,8 @@ def set_payload_for_mail(payload, mail)
       end
 
       def single_send_params(mail)
-        raise MissingTemplateError, "Missing emailId parameter." unless mail.email_id.present?
-        raise MissingRecipientError, "Missing recipient email."  unless mail.to.present?
+        raise MissingTemplateError, "Missing emailId parameter." if mail.email_id.blank?
+        raise MissingRecipientError, "Missing recipient email."  if mail.to.blank?
```

Comment:
Unless you like 'unless' :) .... More seriously, I'm not sure - should unrelated style / formatting changes be in this kind of PR?

Reply:
its our gem now :) I like when things is more readable